### PR TITLE
Grammar

### PR DESCRIPTION
**Pull request by geraldcroes**
### What does this PR do?
Fixes minor grammar issues.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ If you want your users to access some of your microservices from the Internet, y
 - path `domain.com/web` will point the microservice `web` in your private network
 - domain `backoffice.domain.com` will point the microservices `backoffice` in your private network, load-balancing between your multiple instances
 
-But a microservices architecture is dynamic... Services are added, removed, killed or upgraded often, eventually several times a day.
+But a microservices architecture is dynamic... Services are often added, removed, killed, or upgraded, eventually several times a day.
```
Comment (errm) [inline docs/index.md]: Not the fault of this change, but I am not sure about this sentence at all...

> But a microservices architecture is dynamic

Microservices or the architecture is not really the issue here, we are thinking about environments where instances of a service are launched dynamically.

Comment (geraldcroes) [reply docs/index.md]: Maybe something like: 
>But microservices architecture environments are dynamic: Services are often added, 
>removed, killed, or upgraded -- eventually several times a day.

Comment (errm) [reply docs/index.md]: How about:

> Microservices are often deployed dynamic environments where services are added, removed, killed, upgraded or scaled many times a day.

Comment (geraldcroes) [reply docs/index.md]: I think that there's a typo: deployed /in/?

Comment (errm) [reply docs/index.md]: yes, you are correct

**Pull request merged**